### ISSUE TITLE
Can not use flipper template

### ISSUE THREAD

**Issue by realtakahashi** [AstarNetwork/swanky-cli#47]
- version
```
@astar-network/swanky-cli/1.0.0-rc.4 darwin-x64 node-v18.0.0
``` 
<img width="855" alt="image" src="https://user-images.githubusercontent.com/38233656/204525507-d6125c42-bf20-4365-b409-663c48f0d7ee.png">

**Comment by realtakahashi** [comment]
You may already know the solution.
This issue is caused by the absence of test code templates in the following directory:
'node_modules/@astar-network/swanky-cli/node_modules/@astar-network/swanky-templates/dist/templates/contracts/ink/flipper/test'

Creating the "test" directory and "flipper.test.tx.hbs" file here will avoid the error.

**Comment by codespool** [comment]
Thanks! You're right, it's the missing template - will be patched in the next release 👍

**Comment by codespool** [comment]
This is fixed in v1.06.
Note that the flipper contract itself is changed a bit to return a bool value, so it might be inconsistent with original flipper examples.
